pkg/auth/.tmp_gomock_reflect_512816048/prog.go

```diff
@@ -0,0 +1,46 @@
+
+package main
```

Comment:
I guess you did not mean to commit this file

Reply:
Right, removed.